CHANGELOG.md

```diff
@@ -96,7 +96,7 @@ v1.0.0-dev (unreleased)
     Before:
 
     ```go
-    inbound := tchannel.NewInbound(...)
+    inbound := tchannel.NewInboundFromChannel(ch)
```

Comment:
This will eventually, like this penguin, disappear, but for now, must change to match the version.